src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -8624,6 +8632,8 @@
 				BAC5CC0A27BE84DA00094C32 /* HealthCertifiedPersonCoordinator.swift */,
 				BA5B7FCA26458EF400502087 /* HealthCertifiedPersonViewController.swift */,
 				BA5B7FC926458EF400502087 /* HealthCertifiedPersonViewModel.swift */,
+				BAC5CC0C27BEB56600094C32 /* UpdateConsent */,
+				BA8A9FC427BFF788001DBB4B /* UpdateScceeded */,
```

Comment:
I would propose to use the naming from the tech spec, so it would be more `ReissuanceConsent` and `ReissuanceSucceeded` and use everywhere, also in the functions...

Reply:
I did some renaming as suggested